packages/lazy-import/package.json

```diff
@@ -18,9 +18,6 @@
 	"bugs": {
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
-	"engines": {
```

Comment:
I don't think it's needed. My 2 cents are: consumers should always use supported(maintained) Node.js versions already. Unless we use some specific features that are only supported in newer Node.js version, we don't need to enforce it in the `engines` field. WDYT?

Reply:
It's simpler to use `engines` in all packages rather than remember all the exceptions. My hope is that with Node.js 16 becoming the lower-supported LTS version, we can finally get to the point where all `engines` entries look the same for WP packages. I don't mind if that means we remove them completely.

Reply:
+1 for being explicit about version requirements using `engines`. This makes it consistent with Core (it's required there for the build server to work correctly).

Also, if we allow contributors to use Node < 16, then we will run into situations where the lock file format is reverted to the previous version that npm 6.x used.

Reply:
I think we're talking about the `engines` field in the sub-packages, not at the root level `package.json`. I also prefer having version requirements at the root level `package.json` because it helps to keep contributors on the same page and enforce lock file format etc. This doesn't apply to each package's individual `package.json` though, which should only be consumed by users installing from the npm registry. Since they don't need to run builds on npm install, they don't need to pin their Node/npm versions like contributors do.